ext/drreg/drreg.c

```diff
@@ -65,17 +65,53 @@
 
 #define PRE instrlist_meta_preinsert
 
-/* This should be pretty hard to exceed as there aren't this many GPRs */
+/* The number of spills should be pretty hard to exceed as there aren't
+ * that many GPRs. We have a hard-coded max on spill slots that we track.
+ */
 #define MAX_SPILLS (SPILL_SLOT_MAX + 8)
+/* We choose the number of available slots for spilling simds to match their
+ * theoretical max number for a given build.
+ */
+#define MAX_SIMD_SPILLS DR_NUM_SIMD_VECTOR_REGS
```

Comment:
We also need to add + 8 here as well. I can't remember why we had decided to remove it.

Reply:
Perhaps I should remove the magic number, i.e., 8, and add a definition while I am at it.

Reply:
Done